doc/getting-started/installation-instructions/development.rst

```diff
@@ -12,33 +12,66 @@ Prerequisites
 -------------
 
 Knowledge of `Python <https://www.python.org/>`__ (and its `virtual
-environments <https://docs.python.org/3.11/tutorial/venv.html>`__),
+environments <https://docs.python.org/3.12/tutorial/venv.html>`__),
 `git <https://git-scm.com/>`__ and `software
 development <https://xkcd.com/844/>`__ are required.
 
 Some software prerequisites are needed to build the development
 environment. First of your you need a Python version supported by the
-engine. At the moment we recommend Python 3.11, which is the only
-version supported on Windows and macOS.
+engine. At the moment we recommend Python 3.12.
 
-**NB: Python 3.12 are not supported yet, so please do NOT install such
+**NB: Python 3.13 and later versions are not supported yet, so please do NOT install such
 versions**
 
-**NB: Python version below 3.9 are not supported , so please do NOT
+**NB: Python version below 3.11 are not supported , so please do NOT
 install such versions**
 
 Linux
 -----
 
 Please check the documentation of your distribution to install one of
-the supported version of Python. At the moment we recommend Python 3.11
-but it also can used Python 3.9 and 3.10
+the supported version of Python. At the moment we recommend Python version 3.12
+but Python 3.11 can also be used.
 
 macOS
 -----
-
+Python version 3.12 is recommended but Python 3.11 can also be used.
 Please see instructions for the :doc:`Installing the OpenQuake Engine on MacOS <macos>`
 
+Windows
+-----
+
+Python version 3.12 is recommended but Python 3.11 can also be used.
+The use of the official Python installer for Windows is mandatory.
+
```

Comment:
I think we should replace lines 44 and 45 with:

Please install Python version using the official Python installer, using the instructions below. Do **NOT** install using Microsoft Store.